UPDATING.md

```diff
@@ -24,6 +24,14 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+
+#### Change default aws_conn_id in EMR operators
+
+The aws_conn_id was accidently set to 's3_default' instead of 'aws_default' in some of the emr operators in previous 
+versions. This was leading to emr_step_sensors not being able to find their corresponding emr cluster. With the new 
+changes in the emr_add_steps_operator, emr_terminate_job_flow_operator and emr_create_job_flow_operator this issue is 
```

Comment:
You should use the class names, not name of files with classes.

Reply:
I do not know if it would be helpful to add a link to the section in the documentation that discusses how to configure the connection to AWS.
https://airflow.apache.org/en/latest/howto/manage-connections.html#amazon-web-services

Reply:
@mik-laj I changed to the class names and added the link (However, I removed the "en/latest" from the URL as I was not able to open the link you put above).